docs/deployment.md

```diff
@@ -116,13 +116,75 @@ to the service through the load balancer:
 ```bash
 > go run github.com/onosproject/onos-config/cmd/diags/changes \
     -address=config.onosproject.org:443 \
-    -keyPath=deployments/helm/files/certs/tls.key \
-    -certPath=deployments/helm/files/certs/tls.crt
+    -keyPath=deployments/helm/onos-config/files/certs/tls.key \
+    -certPath=deployments/helm/onos-config/files/certs/tls.crt
 ```
 
 The ingress routes requests based on the host header and redirects the HTTP/2
 traffic to provide end-to-end encryption.
 
+### Deploying the device simulator
+
+µONOS Config provides a [device simulator](../tools/test/devicesim/gnmi_user_manual.md)
+for end-to-end testing. As with the µONOS Config app, a [Helm] chart is provided for
+deployment in [Kubernetes]. Each chart instance deploys a single simulated device
+`Pod` and a `Service` through which it can be access. The µONOS Config chart can
```

Comment:
*can access the device*

Reply:
I'm seeing the same as before - remember you have to rebuild the Docker image to see the effect of the changes (from toos/test/devicesim):
```
docker build -t devicesim -f Dockerfile .
```

I'm still getting same output as comment above